### PR TITLE
BKNDLSS-11273 - fixed android tests

### PR DESCRIPTION
**Pull request by AnatoliiStepaniuk**

### CODE REVIEW

File: src/com/backendless/Persistence.java [hunk 1]
```diff
@@ -266,9 +266,12 @@ public void handleFault( BackendlessFault fault )
       throw new IllegalArgumentException( ExceptionMessage.NULL_ENTITY );
 
     Map<String, Object> entityMap = BackendlessSerializer.serializeToMap( entity );
-    String objectId = FootprintsManager.getInstance().getObjectId( entity );
-    if( objectId != null )
-      entityMap.put( "objectId", objectId );
+    if( !entityMap.containsKey( "objectId" ) )
```
Comment (scadgek) [inline src/com/backendless/Persistence.java]: this logic is encapsulated in BackendlessSerializer.serializeToMap(): FootrprintsManager.Inner.putMissingPropsToEntityMap()
check if it works as expected and remove from here

Comment (markpiller) [inline src/com/backendless/Persistence.java]: Use constant - we have it pre-defined somewhere

File: src/com/backendless/Persistence.java [hunk 2]
```diff
@@ -306,9 +309,12 @@ public void handleFault( BackendlessFault fault )
       };
 
       Map<String, Object> entityMap = BackendlessSerializer.serializeToMap( entity );
-      String objectId = FootprintsManager.getInstance().getObjectId( entity );
-      if( objectId != null )
-        entityMap.put( "objectId", objectId );
+      if( !entityMap.containsKey( "objectId" ) )
```
Comment (markpiller) [inline src/com/backendless/Persistence.java]: same comment as above

File: src/com/backendless/Persistence.java (hunk 1 above)
Comment (scadgek) [inline src/com/backendless/Persistence.java]: Persistence.DEFAULT_OBJECT_ID_FIELD

File: src/com/backendless/Persistence.java [hunk 3]
```diff
@@ -266,9 +266,7 @@ public void handleFault( BackendlessFault fault )
       throw new IllegalArgumentException( ExceptionMessage.NULL_ENTITY );
 
     Map<String, Object> entityMap = BackendlessSerializer.serializeToMap( entity );
-    String objectId = FootprintsManager.getInstance().getObjectId( entity );
-    if( objectId != null )
-      entityMap.put( "objectId", objectId );
+    FootprintsManager.getInstance().Inner.putMissingPropsToEntityMap( entity, entityMap );
```
Comment (scadgek) [inline src/com/backendless/Persistence.java]: this line is already executed in serializedToMap()

File: src/com/backendless/Persistence.java [hunk 4]
```diff
@@ -306,9 +304,7 @@ public void handleFault( BackendlessFault fault )
       };
 
       Map<String, Object> entityMap = BackendlessSerializer.serializeToMap( entity );
-      String objectId = FootprintsManager.getInstance().getObjectId( entity );
-      if( objectId != null )
-        entityMap.put( "objectId", objectId );
+      FootprintsManager.getInstance().Inner.putMissingPropsToEntityMap( entity, entityMap );
```
Comment (scadgek) [inline src/com/backendless/Persistence.java]: this line is already executed in serializedToMap()

**Pull request merged**